application/common/config/main.php

```diff
@@ -290,6 +295,8 @@
         'mfaLifetime'                   => Env::get('MFA_LIFETIME', '+2 hours'),
         'mfaManagerBcc'                 => Env::get('MFA_MANAGER_BCC', ''),
         'mfaManagerHelpBcc'             => Env::get('MFA_MANAGER_HELP_BCC', ''),
+        'mfaRecoveryBcc'                => Env::get('MFA_RECOVERY_BCC', ''),
+        'mfaRecoveryHelpBcc'            => Env::get('MFA_RECOVERY_HELP_BCC', ''),
```

Comment:
Likewise, I think these aren't needed. From what I can recall, we never used them for the "Manager" option. My guess is we won't use it with the "Recovery" option either.